server/datastore/mysql/migrations/tables/20240909145426_AddFleetLibraryAppsTable.go

```diff
@@ -0,0 +1,69 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240909145426, Down_20240909145426)
+}
+
+func Up_20240909145426(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE fleet_library_apps (
+	id               int unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+	name             varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+	-- the "full_token" field from homebrew's JSON API response
+	-- see e.g. https://formulae.brew.sh/api/cask/1password.json
+	token            varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+	version          varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+	platform         varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+	installer_url    varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+	filename         varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
Is this the filename of the installer, or of the .app file itself?

Reply:
Good question! That's to support this response payload here: https://github.com/fleetdm/fleet/pull/21801/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64R9219-R9230

So it is the installer's filename (e..g .zip file).